reframe/core/buildsystems.py

```diff
@@ -661,42 +661,52 @@ class EasyBuild(BuildSystem):
     options = fields.TypedField(typ.List[str])
     emit_package = fields.TypedField(bool)
     package_opts = fields.TypedField(typ.Dict[str, str])
+    srcdir = fields.TypedField(str, type(None))
```

Comment:
I would vote against this, since it has a complete different meaning in other build systems. Perhaps the best solution is to call the `emit_build_commands()` after we have changed to the stage directory. Then any build system can be sure that the current working directory is the stage directory.